flow-path/src/test/java/flow/path/ViewStateTest.java

```diff
@@ -1,155 +0,0 @@
-package flow.path;
-
-import android.annotation.SuppressLint;
-import android.os.Bundle;
-import android.os.Parcelable;
-import android.util.SparseArray;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.FrameLayout;
-import flow.Flow;
-import flow.History;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.robolectric.RobolectricTestRunner;
-import org.robolectric.RuntimeEnvironment;
-import org.robolectric.annotation.Config;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-/**
- * This really belongs in Flow proper, but at the moment the only way to get at ViewState
- * is through PathContainer. Oops.
- */
-@RunWith(RobolectricTestRunner.class) // Necessary for functional SparseArray
-@Config(manifest = Config.NONE) //
-public class ViewStateTest {
-  TestScreen able = new TestScreen("able");
-  TestScreen baker = new TestScreen("baker");
-
-  @Test public void stuff() {
-    // Suppose we start off with a flow instance in an portrait-oriented activity;
-    Flow portrait = new Flow(History.single(able));
-    portrait.setDispatcher(new Dispatcher());
-    // We go from able to baker.
-    portrait.set(baker);
-
-    // Pretend we wrote the history to a bundle and read it out again.
-    History history = portrait.getHistory();
-
-    // A new landscape activity instance creates a new flow instance with the "restored" history.
-    Flow landscape = new Flow(history);
```

Comment:
Your espresso test doesn't cover this case, building a new Flow from saved history. I worry about a regression of the fix in https://github.com/square/flow/commit/439536dd4116063548dcf99511eb96105673e9e9